packages/utilities/src/classNamesFunction.ts

```diff
@@ -2,8 +2,11 @@ import { mergeCssSets, IStyleSet, IProcessedStyleSet, Stylesheet } from '@uifabr
 import { IStyleFunctionOrObject } from '@uifabric/merge-styles';
 import { getRTL } from './rtl';
 import { getWindow } from './dom';
+import { StyleFunction } from './styled';
 
 const MAX_CACHE_COUNT = 50;
+const DEFAULT_SPECIFICITY_MULTIPLIER = 5;
```

Comment:
5 seems pretty hefty. Was this really needed?

Reply:
I think we need to tweak this as we go.
Currently, if you purely look at checkbox.scss, i think this number should be 4 to ensure we don't break user. Some component needs lower and some will need higher, depends on how many modifier classes there are for each component. 

It certainly makes the raw css quite ugly/strange. Do you have a perf concern over the size of this number? (from what i read it should make a difference)

Reply:
multiple longer strings injected in bulk. Slight concern but not the end of the world.